src/Library/demos/Status Page/main.css

```diff
@@ -0,0 +1,6 @@
+box {
+  background-color: #f5f5f5;
```

Comment:
Better to use a [named color](https://gnome.pages.gitlab.gnome.org/libadwaita/doc/1-latest/named-colors.html) here.